public/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow: Templates/administration/
```

Comment:
this line can be remove ^^, there are nothing in Templates/administraction/ but if you want to include disallow on admin page , you can just put /administration. value is about route in url, not path in your code ^^,

Reply:
```suggestion
Disallow: /administration/
```